webapp/app/forms/fields.py

```diff
@@ -15,7 +15,21 @@
 from app.forms.validators import ValidElsterCharacterSet
 
 
+class SteuerlotseIntegerField(IntegerField):
+    def __init__(self, label='', validators=None, **kwargs):
+        if kwargs.get('render_kw') and kwargs['render_kw'].get('max_characters'):
+            kwargs['render_kw']['class'] = kwargs['render_kw'].get('class', '') + \
+                                           f' input-width-{kwargs["render_kw"].get("max_characters")}'
+        super().__init__(label, validators, **kwargs)
+
+
 class SteuerlotseStringField(StringField):
+    def __init__(self, label='', validators=None, **kwargs):
+        if kwargs.get('render_kw') and kwargs['render_kw'].get('max_characters'):
+            kwargs['render_kw']['class'] = kwargs['render_kw'].get('class', '') + \
+                                           f' input-width-{kwargs["render_kw"].get("max_characters")}'
+        super().__init__(label, validators, **kwargs)
```

Comment:
I think this works. However, there are now so many different places where we set this, that for my taste setting it in the macro would make a little more sense. I was thinking if it might make sense to have a special macro to set field classes at some point. However, we probably will change how we to that when we settle on a new frontend architecture.

Reply:
Okay, moved it to the macro and thus got rid of the SteuerlotseIntegerField.